### ISSUE TITLE
Error: XDR encoding failure since firefox 83.0

### ISSUE THREAD

**Issue by hacker-h** [xwiki-labs/cryptpad#636]
**Describe the bug**
When loading a cryptpad kanban during loading it shows "Error: XDR encoding failure"

**Where did it happen?**
cryptpad.fr

**To Reproduce**
1. Go to https://cryptpad.fr/kanban/#/3/kanban/edit/XXXXXXXXXXXXXXXXXXXXXXXXXXXX
2. See the error while loading

**Expected behavior**
No error

**Screenshots**
https://i.imgur.com/Lg61qza.png

**Browser (please complete the following information):**
 - OS: Windows 10.0.18363 Build 18363
 - Browser firefox
 - Version 83.0
 - default settings

**Additional context**
It worked with the previous firefox version, so this is most certainly the changing point

**Comment by ansuz** [comment]
We received a report about this yesterday afternoon. You're correct that it was introduced in Firefox 83. We tracked it back to [this line in this commit](https://hg.mozilla.org/mozilla-central/rev/b3ea886b0ae7c2f056fead164116774dab00239b#l14.166).

For anyone that's interested in _why_, it has something to do with loading asm.js (a subset of Javascript which is easily optimized) from a different domain. In our case the _sandbox_ domain which runs all the UI code loads a file containing asm from the _main_ domain. Firefox throws an error/warning, but seems to successfully compile and run the optimized code anyway.

We're working on solving the underlying issue (in case we're doing something we shouldn't) but in the meantime we've simply added [this code](https://github.com/xwiki-labs/cryptpad/commit/d60bdbfba11c4f859be9287014b1cb41f4269600) to prevent the error from being displayed since it doesn't actually interfere with normal behaviour.

Our patch is deployed to CryptPad.fr and has been merged to the `main` branch for any admins who want to update their own instance.
